tests/cupyx_tests/scipy_tests/ndimage_tests/test_filters.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+
+import numpy
+
+import pytest
+
+from cupy import testing
+import cupyx.scipy.ndimage  # NOQA
+
+try:
+    import scipy.ndimage  # NOQA
+except ImportError:
+    pass
+
+
+@testing.parameterize(*testing.product({
+    'shape': [(3, 4), (2, 3, 4), (1, 2, 3, 4)],
+    'ksize': [3, 4],
+    'mode': ['reflect', 'constant', 'nearest', 'mirror', 'wrap'],
+    'cval': [0.0],
+    'origin': [-1, 0, 1],
```

Comment:
Test a sequence.

Reply:
I meant a sequence value, not `None`.
`None` is not specified in NumPy's documentation, actually.

Reply:
`None` is not directly passed to filter functions. When `self.origin = None`, a sequence is created and passed to filter function. This is kind of work-around to support different ndim.
https://github.com/anaruse/cupy/blob/66aaa92fe522581cf6e76bfb6f99adaf5212d160/tests/cupyx_tests/scipy_tests/ndimage_tests/test_filters.py#L33-L36

Reply:
Oh, I overlooked that. Thanks!